File: dynabuffers-java/src/main/kotlin/dynabuffers/api/map/ImplicitDynabuffersMap.kt
package dynabuffers.api.map

import dynabuffers.api.IType

class ImplicitDynabuffersMap(map: Map<String, Any?>,
                             tree: List<IType>,
                             root: IType) : DynabuffersMap(map, tree, root) {
    fun getValue() = get("value")
}

File: dynabuffers-python/dynabuffers/api/map/ImplicitDynabuffersMap.py
from typing import List

from dynabuffers.api.ISerializable import ISerializable
from dynabuffers.api.map.DynabuffersMap import DynabuffersMap


class ImplicitDynabuffersMap(DynabuffersMap):

    def __init__(self, map: dict, tree: List[ISerializable], root: ISerializable):
        super().__init__(map, tree, root)

    def get_value(self):
        return self.get("value")